.github/pr-title-checker-config.json

```
@@ -0,0 +1,10 @@
+{
+	"LABEL": {
+	  "name": "Title needs format :)",
+	  "color": "EEEEEE"
+	},
+	"CHECKS": {
+	  "regexp": "^(((\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])? )|((Regression|Chore|Bump): )).+|(Release [0-9]+\\.[0-9]+.[0-9]+))",
```

what if we use "prefixes" to validate the most common usages, so it is easier to read and maintain, and use "regexp" to validate only edge cases like the second group and the "release 1.2.3"
```suggestion
	"CHECKS": {
		"prefixes": ["[NEW]", "[FIX]", "[IMPROVE]", "[BREAK]", "Regression:", "Chore:", "Bump", "Merge master into develop & Set version"]
		"regexp": "(^\\[[^]+]\\[(ENTERPRISE|APPS)\\]|Release [0-9]+\\.[0-9]+\\.[0-9]+)",```

---

I don't like Snyk PRs :thinking: we might need to fix it's PR titles. fixed the code suggestion.. but feel free to not use it too =)

---

the only reason to support Snyk PRs as they are is that if we want to automate merging that kind of version bumps.

---

I put the expressions to force the user to insert the spaces and ":" using prefix its not possible =/.
I dont think the ability to read needs to be over the capacity to express the rules: it should cover your specification and that should be documented. if the expression to cover that is hard 🤷 there is nothing that we can do about it